src/App/screens/Instructor/components/Nav/index.js

```diff
@@ -5,8 +5,16 @@ import Icon from '../Icon'
 import Logo from '../Logo'
 import NavLink from './components/NavLink'
 
-class Nav extends Component {
+export default class extends Component {
   
+  static propTypes = {
+    pathname: PropTypes.string.isRequired,
+    routes: PropTypes.arrayOf(PropTypes.shape({
+      text: PropTypes.string.isRequired,
+      route: PropTypes.string.isRequired,
+    })).isRequired,
+  }
```

Comment:
This is a class property initializer included in CRA; most likely going out with ES2017. Encouraged by the React team since it lets you keep all class stuff in the class.